server/requirements.txt

```diff
@@ -20,7 +20,7 @@ pandas>=1.0,!=1.1  # pandas 1.1 breaks tests, https://github.com/pandas-dev/pand
 PyYAML>=5.3
 scipy>=1.0
 requests>=2.22.0
-tiledb>=0.5.9,>=0.6.2
+tiledb>=0.5.9,>=0.6.2,<=0.7.1
```

Comment:
Sounds like the problem will be fixed in the next release (coming soon).  Another option is to 
just exclude the problem version, so we don't have to change again:

tiledb>=0.6.2,!=0.7.2